server/neptune/temporalworker/server.go

```diff
@@ -104,19 +104,27 @@ func NewServer(config *Config) (*Server, error) {
 }
 
 func (s Server) Start() error {
-	var wg sync.WaitGroup
-	wg.Add(1)
 	defer s.Logger.Close()
 	defer s.TemporalClient.Close()
+	var wg sync.WaitGroup
+	wg.Add(2)
+
+	go func() {
+		defer wg.Done()
+		w := worker.New(s.TemporalClient, workflows.DeployTaskQueue, worker.Options{})
+		if err := w.Run(worker.InterruptCh()); err != nil {
+			log.Fatalln("unable to start deploy worker", err)
+		}
+	}()
 
 	go func() {
 		defer wg.Done()
-		w := worker.New(s.TemporalClient, workflows.DeployTaskQueue, worker.Options{
+		w := worker.New(s.TemporalClient, workflows.TerraformTaskQueue, worker.Options{
 			// ensures that sessions are preserved on the same worker
 			EnableSessionWorker: true,
 		})
 		if err := w.Run(worker.InterruptCh()); err != nil {
-			log.Fatalln("unable to start worker", err)
+			log.Fatalln("unable to start terraform worker", err)
 		}
 	}()
```

Comment:
hmm seems a bit pointless to run a separate worker process tbh.  it's not like we can scale this separately.  I'm wondering if we should just use the same deploy task queue/worker for now.

Reply:
yeah but you can have our deploy worker as a session worker